src/contentful/ContentfulClient.ts

```diff
@@ -27,9 +27,12 @@ export class ContentfulClient {
         // initialize instance variables
         this.options = options;
 
-        const serverUrl = options.preview
+        let serverUrl = options.preview
             ? ContentfulClient.PREVIEW_URL
             : ContentfulClient.PRODUCTION_URL;
+        if (options.host) {
+            serverUrl = options.host
+        }
```

Comment:
I think it would make more sense to leave the variable unset as a `const`, then write the value if `host` is set, otherwise set the const using the existing logic.